src/page/search/advanced-search/list-summary.js

```diff
@@ -32,6 +49,17 @@ const Summary = {
             groupingKey: undefined
         });
     },
+    _getScopeLabel() {
+        if(this.state && this.state.reference.scopes) {
+            let scopes = this.state.reference.scopes
+            for(let i = 0; i < scopes.length; i++) {
+                if(scopes[i].code === this.props.scope) {
+                    return scopes[i].label;
+                }
+            }
+        }
+        return this.props.scope;
+    },
```

Comment:
@JoanaDalmeida Could you please use a loads function to make this code simplifier ?